src/campaign.js

```diff
@@ -111,7 +121,7 @@ function scheduleNextPush(pushCampaign, now) {
         .then(() => {
           const pushes = pushCampaign.relation('pushes');
           pushes.add(pushStatus);
-          return pushCampaign.save(null, { useMasterKey: true });
+          return pushCampaign.save('nextPush', pushStatus, { useMasterKey: true });
```

Comment:
` save({'nextPush': pushStatus})`

Not sure why it works, how it it possible it works???